spec/controllers/application_controller_spec.rb

```diff
@@ -491,4 +496,32 @@ def append_info_to_payload(payload)
       end
     end
   end
+
+  describe '#sso_logging_info' do
+    subject { get :test_logging }
+
+    let(:user) { create(:user) }
+    let(:session_object) { Session.create(uuid: user.uuid) }
+    let(:request_host) { Settings.hostname }
+    let(:sso_cookie_content) { 'sso_cookie_contents' }
+
+    before do
+      allow(Rails.logger).to receive(:info)
+    end
+
+    context 'when the current user and session object exist' do
+      it 'returns a hash with the user and session info' do
+        expected_result = {
+          user_uuid: user.uuid,
+          sso_cookie_contents: sso_cookie_content, # assuming this method returns a string
+          request_host:
+        }
+        expect(Rails.logger).to receive(:info) do
+          expect(expected_result).to match({ user_uuid: user.uuid, sso_cookie_contents: 'sso_cookie_contents',
+                                             request_host: 'www.example.com' })
```

Comment:
This actually isn't testing anything. All this is doing is comparing the `expected_result` hash with itself. The idea here would be to test if `expected_result` is actually in the logger. Something like:
`expect(Rails.logger).to receive(:info).with(expected_result)`